ui/react-components/components/EditAppointment/EditAppointment.jsx

```diff
@@ -238,7 +238,8 @@ const EditAppointment = props => {
     const checkAndUpdateRecurringAppointments = async (applyForAllInd) => {
         if (isValidRecurringAppointment()) {
             const recurringRequest = getRecurringAppointmentRequest(applyForAllInd);
-            const response = await getRecurringAppointmentsConflicts(recurringRequest);
+            const response = applyForAllInd ? await getRecurringAppointmentsConflicts(recurringRequest) :
```

Comment:
@AlekhyaYalla what does "applyForAllInd" mean ?? I feel variable name is not meaningful..

Reply:
applyForAllIndicator it is. existing variable,

Reply:
hmm..difficult to get the context by name. I think we should handle the name change in the later part of the cards. I am noting it down.